source/developers/cook-books/how-tos/working-with-crafter-studios-api.rst

```diff
@@ -10,71 +10,113 @@ Working with Crafter Studio's APIs
-      < Pragma: no-cache
-      < Expires: 0
-      < Set-Cookie: JSESSIONID=2E114725C82F3EE44ADC04B578A3BE8F; Path=/studio; HttpOnly
-      < Content-Type: application/json;charset=UTF-8
-      < Content-Language: en-US
-      < Transfer-Encoding: chunked
-      < Date: Mon, 22 Jan 2018 21:32:48 GMT
-      <
-      * Connection #0 to host localhost left intact
-      {"username":"admin","first_name":"admin","last_name":"admin","email":"evaladmin@example.com"}
-
-   Note the response returned is a successful 200 status code and the response contains JSON with details for the authenticated user.
-
-   Also found as part of the request is the ``JSESSIONID`` cookie.  You will need this value for all future requests.
+   To use JWT you need to obtain a an access token following these steps:
+
+   1. Login to Crafter Studio
+
+      .. figure:: /_static/images/jwt/global-menu.png
+        :width: 70%
+        :alt: Crafter Studio - Global Menu
+        :align: center
+      |
+
+   2. Open the ``Token Management`` tool
+
+      .. figure:: /_static/images/jwt/token-management.png
+        :width: 70%
+        :alt: Crafter Studio - Token Management
+        :align: center
+      |
+
+   3. Create a new access token
+
+      The only required field for the access token is the label to identify it, however, it is recommended to also
+      set an expiration date to minimize the risk of lost or stolen tokens being used without noticing.
+
+      .. figure:: /_static/images/jwt/create-token.png
+        :width: 70%
+        :alt: Crafter Studio - Create Access Token
+        :align: center
+      |
+
+      Once the expiration date is reached the access token will stop working automatically.
+
+      .. figure:: /_static/images/jwt/create-token-2.png
+        :width: 70%
+        :alt: Crafter Studio - Access Token Expiration
+        :align: center
+      |
+
+   4. Copy the value of the access token
+
+      The value of the access token will not be stored on the server, so it needs to be stored by the user in a safe
+      place as it is impossible to recover it after it is created.
+
+      If an access token is lost or exposed in any way it should be disabled or completeley deleted to avoid any
+      possible use.
+
+      .. figure:: /_static/images/jwt/token-management-2.png
+        :width: 70%
+        :alt: Crafter Studio - Token Management
+        :align: center
+      |
+
+   5. Include the access token with all requests
+
+      Most HTTP clients provide built-in support for access token athentication, in the case of curl you will need
+      to use the header option ``--header`` or ``-H``
+
+      ``curl -H 'Authentication: Bearer <access token>' ...``
+
+   To use HTTP Basic you need to follow these steps:
+
+   1. Enable HTTP Basic auth and restart Crafter Studio
+
+      Update the Crafter Studio configuration override (either from the disk or from the UI) to add the following:
```

Comment:
Maybe say the location of the override file on disk (e.g. `CRAFTER_HOME/data/repos/global/configuration/studio-config-override.yaml`) and also how to access from the UI (e.g. From the global menu -> `Global Config`)